libs/pika/async_mpi/include/pika/async_mpi/transform_mpi.hpp

```diff
@@ -78,48 +76,28 @@ namespace pika::mpi::experimental {
             execution::thread_priority p = use_priority_boost(mode) ?
                 execution::thread_priority::boost :
                 execution::thread_priority::normal;
-            if (inline_req || !pool_exists())
+
+            auto completion_snd = [=](MPI_Request request) -> unique_any_sender<> {
+                if (!inline_com)
+                {
+                    if (request == MPI_REQUEST_NULL)
+                        return transfer_just(default_pool_scheduler(p));
```

Comment:
```suggestion
                    {
                        return transfer_just(default_pool_scheduler(p));
                    }
```

Reply:
I removed the brackets because I was pasting code into the paper and trying to reduce space. I'll put them back